src/test/java/com/synopsys/integration/jira/common/server/JiraServerServiceTestUtility.java

```diff
@@ -86,6 +86,11 @@ public static String getTestProject() {
         return envTestProject != null ? envTestProject : "Test Project";
     }
 
+    public static String getTestCustomFieldId() {
+        String envTestCustomField = System.getenv(JiraTestEnvVars.TEST_CUSTOM_FIELD_ID);
+        return envTestCustomField != null ? envTestCustomField : "customfield_10700";
```

Comment:
Im not sure we even want to include the default "customfield_10700" here.

Reply:
it may be better to put in an assumeTrue() in this getter, so if someone is use this method then the expectation is that they have the environment variable set

Reply:
Ideally if the server is unchanged going forward this should always be "customfield_10700". But I do see the potential issue if its ever pointed  at a different server it wouldn't work correctly. If we  do  not want it here I could continue and pass the null and handle it directly in the test.